amclient/amclientargs.py

```diff
@@ -37,8 +41,18 @@
     help="Relative path to a file in an Archivematica package",
     type=None,
 )
-
-# Reusable option constants (for CLI).
+SPACE_UUID = Arg(
+    name="space_uuid", help="UUID of the space to add the location to", type=None
+)
+SPACE_RELATIVE_PATH = Arg(
+    name="space_relative_path", help="Path relative to a storage space", type=None
+)
+LOCATION_PURPOSE = Arg(
+    name="location_purpose",
+    help='Purpose of storage space location valid choices: "AR", "AS", "CP", "DS", "SD", "SS", "BL", "TS", "RP"',
```

Comment:
(ESL reviewer here!) Would this read better like `Purpose of storage space location. Valid choices: ...`?

Reply:
Absolutely! I'll make that change then rebase and merge. Great spot!